package.json

```diff
@@ -26,7 +26,15 @@
     "extends": [
       "react-app",
       "react-app/jest"
-    ]
+    ],
+    "@typescript-eslint/no-unused-vars": [
+      "warn",
+      {
+        "args": "all",
+        "argsIgnorePattern": "^_"
+      }
+    ],
+    "react/function-component-definition": "warn"
```

Comment:
I forgot to mention that these should be wrapped under a `rules` property, otherwise the config is invalid. It does not even compile the app. Once you get it running, you will see warnings elsewhere in code. Please handle them as well.

Reply:
done